tests/simulation/physics/test_sparams.py

```
@@ -0,0 +1,204 @@
+    objects, constraints, config = _build_waveguide_sparams()
+
+    key = jax.random.PRNGKey(0)
+    obj_container, arrays, _params, config, _ = fdtdx.place_objects(
+        object_list=objects,
+        config=config,
+        constraints=constraints,
+        key=key,
+    )
+    arrays = fdtdx.extend_material_to_pml(objects=obj_container, arrays=arrays)
+
+    result, detector_states = fdtdx.calculate_sparam(
```

Ruff (0.15.10)

[warning] Unpacked variable `detector_states` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)